cis-aks-controls/CIS-5.4.1.json

```diff
@@ -11,8 +11,10 @@
     "attributes": {
         "armoBuiltin": true
     },
-    "rulesNames": [],
-    "baseScore": 8,
+    "rulesNames": [
+        "rule-manual"
+    ],
+    "baseScore": 0,
```

Comment:
please pull from CIS-AKS branch to update the score

Reply:
fixed